app/controllers/api/v1/rooms_controller.rb

```diff
@@ -22,9 +22,9 @@ class RoomsController < ApiController
       # GET /api/v1/rooms.json
       def index
         # Return the rooms that belong to current user
-        user_rooms = current_user.rooms.search(params[:search])
+        user_rooms = current_user.rooms.order(online: :desc).search(params[:search])
 
-        shared_rooms = current_user.shared_rooms.search(params[:search]).map do |room|
+        shared_rooms = current_user.shared_rooms.order(online: :desc).search(params[:search]).map do |room|
```

Comment:
We should be able to order by more than one value - it  should be `online: :desc` first and then `created_at: :desc` next

Reply:
I think I'd add `last_session: :desc` as second sorting parameter then.